unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -954,7 +959,7 @@ mod tests {
 
     #[test]
     fn handles_dates_with_truncated_milliseconds() {
-        let rule = "current_time > 2022-01-29T13:00:00Z";
+        let rule = "context[\"cutoff\"] > 2022-01-29T13:00:00Z";
```

Comment:
This test was always broken and was passing because of a short circuit. This makes the test green again